darts/tests/test_backtesting.py

```diff
@@ -206,8 +208,6 @@ def test_backtest_regression(self):
         self.assertGreater(score, 0.9)
 
     def test_gridsearch(self):
-
-        np.random.seed(1)
```

Comment:
Could it be that removing this makes the creation of the random walk time series below non-deterministic?